lib/surface/components/form/error_tag.ex

```diff
@@ -0,0 +1,81 @@
+defmodule Surface.Components.Form.ErrorTag do
+  @moduledoc """
+  An error tag inspired by `error_tag/3` that ships with `mix phx.new`
+  in `MyAppWeb.ErrorHelpers`.
+
+  Renders error messages if there are any about the given field.
+
+  ## Examples
+
+  ```
+  <ErrorTag field="password" />
+  ```
+  """
+
+  use Surface.Components.Form.Input
```

Comment:
I'm not 100% sure that we want this line. I think we do want `form`, `field`, and `class` props though. And `InputContext`.

Reply:
Yup. We don't want that line. It should only have it in components that use `<input ...>`. In this case, it's better to define those properties explicitly.

Reply:
Perfect, thanks for the feedback. Fixed that.